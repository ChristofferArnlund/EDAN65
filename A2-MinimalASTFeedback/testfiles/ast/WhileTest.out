Program
  List
    Func
      Name ID="a"
      List
      List
        IdDecl
          Name ID="x"
          Opt
        While
          Ge
            Name ID="x"
            Numeral NUMERAL="4"
          List
            IdDecl
              Name ID="y"
              Opt
        While
          Numeral NUMERAL="1"
          List
