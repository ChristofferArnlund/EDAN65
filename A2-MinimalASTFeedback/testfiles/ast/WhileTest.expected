Program
  List
    Func
      IdDecl
        Name ID="a"
        Opt
      List
      StmtBlock
        List
          IdDecl
            Name ID="x"
            Opt
          While
            Ge
              IdUse
                Name ID="x"
              Numeral NUMERAL="4"
            StmtBlock
              List
                IdDecl
                  Name ID="y"
                  Opt
          While
            Numeral NUMERAL="1"
            StmtBlock
              List
